### PR TITLE
fix: fix export when an asset has a slash

### PR DESCRIPTION
**Pull request by romainrousseaux**
for example, `as/set.png` will be exported as : 

- labels/
  - as /
    - set.png

### CODE REVIEW

File: src/kili/services/export/format/voc/__init__.py [hunk 1]
```diff
@@ -121,7 +122,10 @@ def _process_asset(
             json_response, width, height, parameters, valid_jobs
         )
         xml_filename = f'{asset["externalId"]}.xml'
-        with open(labels_folder / xml_filename, "wb") as fout:
+        filepath = labels_folder / xml_filename
+        print(filepath)
```
Comment (Jonas1312) [inline src/kili/services/export/format/voc/__init__.py:126]: remove prints (use warnings, or logs if need to write to console)

File: tests/unit/services/export/test_geojson.py [hunk 2]
```diff
@@ -64,3 +64,39 @@ def test_kili_export_labels_geojson(mocker: pytest_mock.MockerFixture):
 
     assert output["type"] == "FeatureCollection"
     assert len(output["features"]) == 5  # 5 annotations in geotiff_image_project_assets.json
+
+
+def test_process_asset_with_external_id_containing_slash():
+    with TemporaryDirectory() as f:
```
Comment (Jonas1312) [inline tests/unit/services/export/test_geojson.py:70]: you can get a temp dir using pytest fixture: https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture

**Review by Jonas1312 — COMMENTED** [review]
thanks for  the PR

can you fix the pyright error too?

**Review by PierreLeveau — APPROVED** [review]

**Review by Jonas1312 — APPROVED** [review]

**Comment by Jonas1312** [comment]
wait for e2e tests to pass before merging: https://github.com/kili-technology/kili-python-sdk/actions/runs/7062228310

**Pull request merged**